### PR TITLE
create json file with test

### PR DESCRIPTION
**Pull request by jacquigiese**

### CODE REVIEW

File: CsvParser/csvParser.go [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 	"io"
-	"time"
-  	"log"
+	"log"
 	"os"
 	"strconv"
-	"encoding/csv"
 	"strings"
+	"time"
+
```
Comment (hmc-elephant-s18) [inline CsvParser/csvParser.go]: Is there a reason you left this line blank? I would get rid of it

File: CsvParser/csvParser.go [hunk 2]
```diff
@@ -39,44 +41,43 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
...
-	for i := 0 ; i < 100; i++{
-		
+	for i := 0; i < 100; i++ {
+
 		rec, err = reader.Read()
-		if err != nil{
-			if err == io.EOF{
+		if err != nil {
+			if err == io.EOF {
 				break
```
Comment (hmc-elephant-s18) [inline CsvParser/csvParser.go]: good fixes x

File: CsvParser/csvParser.go [hunk 3]
```diff
@@ -85,4 +86,45 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
...
+		bpStart, bpEnd := arr[i].GetBadPeriod()
+		s := arr[i].GetSeverity()
+		ta := arr[i].GetTestsAffected()
+		gpi := arr[i].GetGoodPeriodInfo()
+		bpi := arr[i].GetBadPeriodInfo()
+		ii := arr[i].GetIncidentInfo()
+		inc := incident.IncidentData{gpStart, gpEnd, bpStart, bpEnd, s, ta, gpi, bpi, ii}
+		objs[i] = inc
+	}
+	b, err := json.Marshal(objs)
```
Comment (hmc-elephant-s18) [inline CsvParser/csvParser.go]: try coming up with variable names that are longer than one or two characters

File: CsvParser/csvParser.go (hunk 1 above)
Comment (jacquigiese) [reply CsvParser/csvParser.go]: Blank line removed

File: CsvParser/csvParser.go (hunk 3 above)
Comment (jacquigiese) [reply CsvParser/csvParser.go]: Changed var names (except for n because it is only for file write errors anyways)

File: CsvParser/csvParser.go [hunk 4]
```diff
@@ -85,4 +86,47 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
...
+	for i := 0; i < numObjects; i++ {
+		gpStart, gpEnd := arr[i].GetGoodPeriod()
+		bpStart, bpEnd := arr[i].GetBadPeriod()
+		gMetric := arr[i].GetGoodMetric()
+		bMetric := arr[i].GetBadMetric()
+		severity := arr[i].GetSeverity()
+		testsAffected := arr[i].GetTestsAffected()
+		gpInfo := arr[i].GetGoodPeriodInfo()
+		bpInfo := arr[i].GetBadPeriodInfo()
+		iInfo := arr[i].GetIncidentInfo()
```
Comment (hmc-elephant-s18) [inline CsvParser/csvParser.go]: may be space out these into like 3 three chunks but it is not that important at all

File: CsvParser/csvParser.go [hunk 5]
```diff
@@ -85,4 +86,47 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
...
+		inc := incident.IncidentData{gpStart, gpEnd, bpStart, bpEnd, gMetric, bMetric, severity, testsAffected, gpInfo, bpInfo, iInfo}
+		objs[i] = inc
+	}
+	bytes, err := json.Marshal(objs)
+	n, err := f.Write(bytes)
+
+	if err != nil {
+		fmt.Println(n)
```
Comment (hmc-elephant-s18) [inline CsvParser/csvParser.go]: get rid of the print statements

File: incident_viewer_demo/incident/incident.go [hunk 6]
```diff
@@ -44,38 +92,49 @@ testsAffected int){
 	i.numTestsAffected = testsAffected
 }
 
-func (i *DefaultIncident) getGoodPeriod() (time.Time, time.Time) {
+//**************************************//
+//    		GETTER METHODS
+//**************************************//
```
Comment (hmc-elephant-s18) [inline incident_viewer_demo/incident/incident.go]: add a space here

File: incident_viewer_demo/incident/incident.go [hunk 7]
```diff
@@ -26,13 +45,42 @@ type DefaultIncident struct {
...
-func (i *DefaultIncident) Init ( goodTimeStart time.Time, goodTimeEnd time.Time,
-badTimeStart time.Time, 
-badTimeEnd time.Time,
-avgDSGood float64,
-avgDSBad float64,
-severity float64,
-testsAffected int){
+//**************************************//
+//    		CONSTRUCTORS
+//**************************************//
```
Comment (hmc-elephant-s18) [inline incident_viewer_demo/incident/incident.go]: may be add a space here

File: CsvParser/csvParser.go (hunk 5 above)
Comment (jacquigiese) [reply CsvParser/csvParser.go]: Changed to log

**Pull request merged**